### PR TITLE
Add updated at column

### PR DESCRIPTION
**Pull request by FelixMalfait**
Introduced `updatedAt` column. and fix an existing bug where the field edition page was crashing because we were trying to compute Date('now') (param coming from the default value)

### CODE REVIEW

**Review by Weiko — APPROVED** [review]
LGTM

File: packages/twenty-front/src/modules/settings/data-model/fields/preview/utils/getComputedDefaultValue.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// Some default value are special keywords such as 'uuid' or 'now'
+// that are mapped to a function that generates a value on the backend.
+// (see serializeFunctionDefaultValue.ts on the server)
+// We need to do a similar mapping on the frontend
+
+import { v4 } from 'uuid';
+
+export const getComputedDefaultValue = (defaultValue?: any) => {
```
Comment (charlesBochet) [inline packages/twenty-front/src/modules/settings/data-model/fields/preview/utils/getComputedDefaultValue.ts:8]: looks similar to generateEmptyFieldValue, I think we should re-use :)

Comment (charlesBochet) [inline packages/twenty-front/src/modules/settings/data-model/fields/preview/utils/getComputedDefaultValue.ts:8]: any typing?

Comment (FelixMalfait) [reply packages/twenty-front/src/modules/settings/data-model/fields/preview/utils/getComputedDefaultValue.ts:8]: As discussed in DM I don't think it's similar

Comment (FelixMalfait) [reply packages/twenty-front/src/modules/settings/data-model/fields/preview/utils/getComputedDefaultValue.ts:8]: defaultValue is never typed in the codebase today. What typing would you suggest?

**Pull request merged**